### DISCUSSION TITLE
Stratis-vdo packages not found

### DISCUSSION THREAD

**Discussion by dvarounis** [rdbreak/rhcsa8env#41] [Q&A] [answered]
Hello,
First of all many thanks for your lab!
My is question is where we can find the packages of stratis and VDO which are in scope of the RHCSA 8 certification.

Kind Regards!
Dimos Varounis

**Comment by dvarounis** [comment, marked as answer]
Sorry, stupid question...The repos should be declared and enabled.

**Reply by Pazze12** [reply]
How?
